bindings/python/iota_sdk/client/client.py

```diff
@@ -321,6 +322,13 @@ def get_node(self) -> Dict[str, Any]:
         """
         return self._call_method('getNode')
 
+    def get_info(self) -> NodeInfoWrapper:
```

Comment:
We already have this method in `bindings/python/iota_sdk/client/_node_core_api.py` ?

Reply:
Yes and even without `print(f'{result}')` :trollface:

Reply:
i swear, i looked everywhere and used the search :D it didnt show!!